lib/app/components/organisms/sent_message.dart

```diff
@@ -12,6 +12,17 @@ class SentMessage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return ChatBubble(text: text, isSender: true);
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.end,
+      children: [
+        ChatBubble(
```

Comment:
```suggestion
        CustomChatBubble(
```
`CustomChatBubble`のクラス名にしましょう

Reply:
ご指摘の通り修正しました！